### PR TITLE
Configure ephemeral storage.  

### PR DESCRIPTION
**Pull request by at88mph**
Relies on https://github.com/opencadc/science-platform/pull/676.

Includes clean up and better documentation.  Also includes a minor version change as it will be incompatible.

Merge `main` once #676 is merged.

### CODE REVIEW

**Review by SharonGoliath — DISMISSED** [review]

**Comment by brianmajor** [comment]
Hi @at88mph - What story is this PR for?

**Comment by at88mph** [comment]
@brianmajor It's for [CADC-13545](https://cadc-ccda.atlassian.net/browse/CADC-13545).

[CADC-13545]: https://cadc-ccda.atlassian.net/browse/CADC-13545?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

File: deployment/helm/skaha/sample-local-values.yaml [hunk 1]
```diff
@@ -7,28 +7,33 @@ deployment:
     imagePullPolicy: IfNotPresent
     # Space delimited list of allowed Image Registry hosts.  These hosts should match the hosts in the User Session images.
     registryHosts: "images.canfar.net"
+
     # Root of shared storage
-    skahaTld: "/arc"
+    skahaTld: "/cavern"
     
-    # Change this to enable the GPU 
+    # Change this to enable a GPU, if one is present on a Node
```
Comment (brianmajor) [inline deployment/helm/skaha/sample-local-values.yaml]:
```suggestion
    # When set to 'true' this flag will enable GPU node scheduling.
```

File: deployment/helm/README.md [hunk 2]
```diff
@@ -248,45 +248,55 @@ deployment:
     # - containerPort: 5555
     #   protocol: TCP
 
-    # Optional.  Rename the main root folder to something else.  For existing installs, this can be
-    # omitted.
-    # skahaTld: "/arc"
+    # Set the top-level-directory name that gets mounted at the root.
+    # skahaTld: "/cavern"
```
Comment (brianmajor) [inline deployment/helm/README.md:252]: I know it's not related to this PR, but I wonder if cavernRoot would have been a better name.
But also, what is this used for?  I know we're trying to decouple skaha and storage altogether, but I guess this field is used in the few references remaining?

File: deployment/helm/skaha/sample-local-values.yaml [hunk 3]
```diff
@@ -7,28 +7,33 @@ deployment:
     imagePullPolicy: IfNotPresent
     # Space delimited list of allowed Image Registry hosts.  These hosts should match the hosts in the User Session images.
     registryHosts: "images.canfar.net"
+
     # Root of shared storage
-    skahaTld: "/arc"
+    skahaTld: "/cavern"
```
Comment (brianmajor) [inline deployment/helm/skaha/sample-local-values.yaml:12]: Same comment here... cavernRoot?

File: deployment/helm/skaha/sample-local-values.yaml [hunk 4]
```diff
@@ -7,28 +7,33 @@ deployment:
...
-    # - name: CATALINA_OPTS
-    #   value: "-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=0.0.0.0:5555"
-    # - name: JAVA_OPTS
-    #   value: "-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=0.0.0.0:5555"
+    # Session settings.
+    sessions:
+      expirySeconds: "345600" # Four days
+      maxCount: "2"  # Users may have 2 interactive sessions at once
+      minEphemeralStorage: "10Gi" # Initial request for ephemeral storage (scratch disk space)
+      maxEphemeralStorage: "100Gi" # Max expansion for ephemeral storage (scratch disk space)
```
Comment (brianmajor) [inline deployment/helm/skaha/sample-local-values.yaml:36]: This area is nice and tidy.

**Review by brianmajor — CHANGES_REQUESTED** [review]
Some comments and suggestions, most not actually related to the intent of the PR!

**Review by brianmajor — APPROVED** [review]

**Pull request merged**